app/connectors/CustomsDeclareExportsConnector.scala

```diff
@@ -20,29 +20,27 @@ import config.AppConfig
 import connectors.CustomsDeclareExportsConnector.toXml
 import connectors.exchange.ExportsDeclarationExchange
 import javax.inject.{Inject, Singleton}
+import models._
 import models.declaration.notifications.Notification
 import models.declaration.submissions.Submission
 import models.requests.CancellationStatus
-import models._
 import play.api.Logger
 import play.api.http.{ContentTypes, HeaderNames}
 import play.api.libs.json.{Json, Writes}
 import play.api.mvc.Codec
-import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
 import uk.gov.hmrc.play.bootstrap.http.HttpClient
 import wco.datamodel.wco.documentmetadata_dms._2.MetaData
 
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class CustomsDeclareExportsConnector @Inject()(appConfig: AppConfig, httpClient: HttpClient) {
-
   private val logger = Logger(this.getClass)
 
-  private def logPayload[T](prefix: String, payload: T)(implicit wts: Writes[T]): T = {
-    Logger.debug(s"$prefix: ${Json.toJson(payload)}")
-    payload
-  }
+  def deleteDraftDeclaration(id: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] =
+    httpClient
+      .DELETE(s"${appConfig.customsDeclareExports}${appConfig.declarationsV2}/$id")
```

Comment:
I know you're doing what the other connector methods do, but should we put the `${appConfig.customsDeclareExports}${appConfig.declarationsV2}` value out somewhere in this class?

Reply:
This has been addressed.